lms/static/js/learner_dashboard/views/program_details_sidebar_view.js

```diff
@@ -8,6 +8,7 @@ import StringUtils from 'edx-ui-toolkit/js/utils/string-utils';
 import CertificateView from './certificate_list_view';
 import ProgramProgressView from './progress_circle_view';
 
+import arrowUprightIcon from '../../../images/arrow-upright-icon.svg';
```

Comment:
I was instructed by #edx-fedx that the approach for icons, when we can't import them directly from Paragon, is to try to use `material-icons` NPM package (https://github.com/material-icons/material-icons/tree/master/svg). 

If this can't be used, then resort to using an SVG like you did here.

Reply:
We would need to add that npm package as its not already here in this repo, if that's fine then I can do that,
Currently I used svgs as files to avoid an extra dependency. What's your take on this?

Reply:
Sorry for the delay, that's true, let's just use the SVG file like you already have here. Thanks for checking!